src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -302,6 +324,34 @@ else if (cmdLine.hasOption(OPTION_C_NAME)) {
     }
 
     /**
+     * Verifies threads number CLI parameter value.
+     * @param cmdLine a command line
+     * @param result a resulting list of errors
+     * @param cliParameterName a CLI parameter name
+     * @param mustBeGreaterThanZeroMessage a message which should be reported
+     *                                     if the number of threads is less than or equal to zero
+     * @param invalidNumberMessage a message which should be reported if the passed value
+     *                             is not a valid number
+     */
+    private static void verifyThreadsNumberParameter(CommandLine cmdLine, List<String> result,
+        String cliParameterName, String mustBeGreaterThanZeroMessage,
+        String invalidNumberMessage) {
+        if (cmdLine.hasOption(cliParameterName)) {
+            final String checkerThreadsNumberStr =
+                cmdLine.getOptionValue(cliParameterName);
+            try {
+                final int checkerThreadsNumber = Integer.parseInt(checkerThreadsNumberStr);
+                if (checkerThreadsNumber < 1) {
+                    result.add(mustBeGreaterThanZeroMessage);
+                }
+            }
+            catch (NumberFormatException ignored) {
+                result.add(invalidNumberMessage);
+            }
```

Comment:
@romani I agree.
`isInt` should be in `CommonUtils` with other similar methods.